README.md

```diff
@@ -110,3 +110,8 @@ Users are currently unable to filter the Guided Answers on Products or Component
 ## Known Issues
 
 https://github.com/SAP/guided-answers-extension/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+## Design Sources
+
+Figma file (Read Only) - https://www.figma.com/file/XSZw3w81MYY0QPe1BoUcBO/Guided-Answers?node-id=1%3A4211&t=vVdqU90dr8pTwJQP-1 
+General Design Guidelines - https://wiki.one.int.sap/wiki/display/fioritools/SAP+Fiori+tools+Home
```

Comment:
@gpeuc, you'll need a logon to access this link. Sure we should publish this here?

Reply:
@Klaus-Keller Figma link is public read-only. No login necessary. You might be right about Wiki, it seems it requires SAP SSO. Can you remove it then please?

Reply:
@gpeuc, yes, I was talking about the wiki link only. I'll see if I can do changes, not sure as it is your fork you created this PR from.